lib/tests/change-federation.js

```diff
@@ -0,0 +1,287 @@
+        const regtestFedChangeAuthorizer3PrivateKey = REGTEST_FEDERATION_CHANGE_PRIVATE_KEYS[2];
+
+        let fedChangeAuthorizer1Address;
+        let fedChangeAuthorizer2Address;
+        let fedChangeAuthorizer3Address;
+        let initialFederationPublicKeys;
+        let newFederationBtcPublicKeys;
+        let initialFederationAddress;
+        let commitFederationEvent;
+        let expectedNewFederationAddress;
+        let commitFederationCreationBlockNumber;
+        let expectedNewFederationErpRedeemScript;
+        let newFederationPublicKeys;
+        let btcTxHelper;
+    
+        before(async () => {
+
+            await startNewFederationNodes(newFederationConfig.members);
+
+            rskTxHelpers = getRskTransactionHelpers();
+            rskTxHelper = rskTxHelpers[0];
+            bridgeTxParser = new BridgeTransactionParser(rskTxHelper.getClient());
+            btcTxHelper = getBtcClient();
+
+            await fundNewFederators(rskTxHelper, newFederationConfig.members);
+
+            await rskUtils.waitForSync(rskTxHelpers);
+
+            // Import the private keys of the federation change authorizers.
+            fedChangeAuthorizer1Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer1PrivateKey);
+            fedChangeAuthorizer2Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer2PrivateKey);
+            fedChangeAuthorizer3Address = await rskTxHelper.importAccount(regtestFedChangeAuthorizer3PrivateKey);
+
+            // Send some funds to the federation change authorizers to pay for transaction fees while voting.
+            await rskUtils.sendFromCow(rskTxHelper, fedChangeAuthorizer1Address, btcToWeis(0.1));
+            await rskUtils.sendFromCow(rskTxHelper, fedChangeAuthorizer2Address, btcToWeis(0.1));
+            await rskUtils.sendFromCow(rskTxHelper, fedChangeAuthorizer3Address, btcToWeis(0.1));
+
+            bridge = getBridge(rskTxHelper.getClient());
+
+            initialFederationPublicKeys = await getCurrentFederationKeys(bridge);
+            newFederationPublicKeys = getNewFederationPublicKeysFromNewFederationConfig(newFederationConfig);
+            newFederationBtcPublicKeys = newFederationPublicKeys.map(federator => federator[KEY_TYPE_BTC]);
+
+            const expectedNewFederationErpRedeemScriptBuffer = redeemScriptParser.getP2shErpRedeemScript(newFederationBtcPublicKeys.map(key => removePrefix0x(key)), ERP_PUBKEYS, ERP_CSV_VALUE);
+            expectedNewFederationErpRedeemScript = expectedNewFederationErpRedeemScriptBuffer.toString('hex');
+            expectedNewFederationAddress = redeemScriptParser.getAddressFromRedeemScript('REGTEST', expectedNewFederationErpRedeemScriptBuffer);
+
+            initialFederationAddress = await bridge.methods.getFederationAddress().call();
+
+            await btcTxHelper.importAddress(initialFederationAddress, 'federations');
+            await btcTxHelper.importAddress(expectedNewFederationAddress, 'federations');
```

Comment:
Is there not prob with both having the same label?

Reply:
No, because they are both part of the federation. At least this is how they are defined in the old `04_00_02-fedchange.js` file.